sklearn/utils/tests/test_random.py

```diff
@@ -178,3 +181,25 @@ def test_random_choice_csc_errors():
     class_probabilities = [np.array([0.5, 0.6]), np.array([0.6, 0.1, 0.3])]
     assert_raises(ValueError, random_choice_csc, 4, classes,
                   class_probabilities, 1)
+
```

Comment:
can you please add a test that checks that if we log and compute a histogram the histogram is approximately uniform?